Movies-Selection/Movies-Selection/ViewController/MovieModule/View/MovieTableView.swift

```diff
@@ -0,0 +1,126 @@
+// MovieTableView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Protocol -для View
+protocol MovieViewProtocol: AnyObject {
```

Comment:
```suggestion
protocol MovieViewProtocol: AnyObject {
```
это лучше в презентуете оставить